docs/COMPILATION.MD

```diff
@@ -188,8 +188,33 @@ make
 Once set up you can run the GUI interface from the terminal `./ccextractorGUI`
 
 ## Windows
+Note: Following screenshots and steps are written on basis of visualstudio version: 2017
 
-Open the windows/ccextractor.sln file with Visual Studio (2015 at least), and build it.
+1.Open `windows/` directory to locate `ccextractor.vcxproj`,  `ccextractorGUI.vcxproj` (blue arrows) and `ccextractor.sln` (red arrow).
+
+![Project Files](img/projectFiles.png)
+
+
+
```

Comment:
I know these blank lines don't render in markdown, but why keep them unnecessarily? :)